### PR TITLE
zh: sync update deployment/deployment-models doc to latest

### PR DESCRIPTION
**Pull request by shuaijinchao**
Please provide a description for what this PR is for.
sync update document content/zh/docs/ops/deployment/deployment-models/index.md to latest

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by shuaijinchao** [comment]
@kebe7jun please review

File: content/zh/docs/ops/deployment/deployment-models/index.md [hunk 1]
```diff
@@ -98,6 +98,16 @@ Kubernetes 在每个集群都默认配置此行为，这有助于限制由错误
 您可以根据[网络](#network-models)和云提供商所支持的选项来配置集群间通信。
 例如，若两个集群位于同一基础网络，则可以通过简单地配置防火墙规则来启用跨集群通信。
 
+<!--
```
Comment (kebe7jun) [inline content/zh/docs/ops/deployment/deployment-models/index.md:101]: needs remove this?

Comment (shuaijinchao) [reply content/zh/docs/ops/deployment/deployment-models/index.md:101]: I think adding the content of the original English document through annotations can help the next person who changes the document to better track the changes to the English document, what do you think?

Comment (Xunzhuo) [reply content/zh/docs/ops/deployment/deployment-models/index.md:101]: I'm -1 for this, in zh docs, if we keep the english docs, it means that we should keep updating both en and zh docs if any changes made in en docs. For a comparison purpose, referring to the original en docs is the most direct/easy way:)

Comment (shuaijinchao) [reply content/zh/docs/ops/deployment/deployment-models/index.md:101]: shouldn't zh docs track changes to en docs? using the method of refer to the original en docs may cause the content to be uneven and confusing to readers. Does this lose the meaning of document localization?

Comment (Xunzhuo) [reply content/zh/docs/ops/deployment/deployment-models/index.md:101]: We should track the changes from en docs, but remaining the en doc in zh is not a good way.

Comment (shuaijinchao) [reply content/zh/docs/ops/deployment/deployment-models/index.md]: got it thx~, #10951 has done this update, I will close this PR.

**Pull request closed without merging**